javaslang-benchmark/pom.xml

```diff
@@ -27,6 +27,16 @@
             <artifactId>javaslang</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <!--<dependency>-->
+        <!--<groupId>org.scala-lang</groupId>-->
+        <!--<artifactId>scala-library</artifactId>-->
+        <!--<version>2.12.0-M4</version>-->
+        <!--</dependency>-->
```

Comment:
maybe I missed the discussion on this, but why you're leave commented dependency?